packages/app-store/zohocrm/api/_getAdd.ts

```diff
@@ -13,7 +14,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   if (!client_id) return res.status(400).json({ message: "zohocrm client id missing." });
   const state = encodeOAuthState(req);
 
-  const redirectUri = WEBAPP_URL + "/api/integrations/zohocrm/callback";
-  const url = `https://accounts.zoho.com/oauth/v2/auth?scope=ZohoCRM.modules.ALL,ZohoCRM.users.READ,AaaServer.profile.READ&client_id=${client_id}&response_type=code&access_type=offline&redirect_uri=${redirectUri}&state=${state}`;
+  const params = {
+    client_id,
+    response_type: "code",
+    redirect_uri: WEBAPP_URL + "/api/integrations/zohocrm/callback",
+    scope: ["ZohoCRM.modules.ALL", "ZohoCRM.users.READ", "AaaServer.profile.READ"],
+    access_type: "offline",
+    state,
+    prompt: "consent",
+  };
```

Comment:
It was hard to debug what was wrong, so let's separate the URL params. Credit to @murtajaziad in https://github.com/calcom/cal.com/pull/10429

Reply:
I think just `prompt` was missing. Earlier it was working fine, maybe Zoho changed something?